pkg/config/config.go

```diff
@@ -52,6 +55,13 @@ func (c Controller) Validate() error {
 		errs = append(errs, "newRelic.licenseKey cannot be blank")
 	}
 
```

Comment:
i don't think we need to use `lo` here - we can just use our faux-go-enum types as a validation

```
	switch c.Buildkit.Compression {
	case CompressionGzip, CompressionStargz, CompressionZstd:
		break
	default:
		// force gzip as the default if not otherwise set
		c.Buildkit.Compression = CompressionGzip
		CompressionMethod = CompressionGzip
	}
```

Reply:
resolved